ironfish/src/primitives/blockheader.ts

```diff
@@ -203,6 +206,9 @@ export class BlockHeader {
     bw.writeBigU256BE(this.target.asBigInt())
     bw.writeU64(this.timestamp.getTime())
     bw.writeBytes(this.graffiti)
+    if (this.stateRoot) {
```

Comment:
There's a constant `180` in this function that this will probably run afoul of -- might need to search around and see if we have the block header size hardcoded anywhere 😬

Reply:
Good call! I noticed that we have this `180` in the `blockHasher` code, but I